airflow/providers/cncf/kubernetes/callbacks.py

```diff
@@ -41,15 +41,15 @@ class KubernetesPodOperatorCallback:
 
     @staticmethod
     def on_sync_client_creation(*, client: k8s.CoreV1Api, **kwargs) -> None:
-        """Callback method called after creating the sync client.
+        """Call this after creating the sync client.
 
         :param client: the created `kubernetes.client.CoreV1Api` client.
         """
         pass
 
     @staticmethod
     def on_pod_creation(*, pod: k8s.V1Pod, client: client_type, mode: str, **kwargs) -> None:
-        """Callback method called after creating the pod.
+        """Call this after creating the pod.
```

Comment:
I guess these might be more close to the original docstring title:
- `Invoke the callback method after the pod has been created.`
- `Invoke the callback method following the creation of the pod.`

Reply:
I see, good suggestion. Thanks.
what about using `Invoke this callback after the pod has been created.`

Reply:
sounds good

Reply:
done, pushed new changes